src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Home</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+  </head>
+  <body>
+    <div class="todo-list">
+      <div class="head-section">
+        <p class="head-title">Today's To Do</p>
+        <img src="../assets/reset.png" alt="" class="head-icon">
```

Comment:
- [ ] Double-check the style of this icon, this happens when I run your app:

![image](https://user-images.githubusercontent.com/60631456/125632258-c599e4b6-3d94-446c-8c63-a1827c1687e1.png)

Target style:

![image](https://user-images.githubusercontent.com/60631456/125632326-038a289b-e4a1-46bd-8ab5-28b2a374b343.png)